docs/Contributing/API-for-contributors.md

```diff
@@ -2222,6 +2222,7 @@ Device-authenticated routes are routes used by the Fleet Desktop application. Un
 - [Get device's transparency URL](#get-devices-transparency-url)
 - [Download device's MDM manual enrollment profile](#download-devices-mdm-manual-enrollment-profile)
 - [Migrate device to Fleet from another MDM solution](#migrate-device-to-fleet-from-another-mdm-solution)
+- [Trigger Filevault key escrow](#trigger-filevault-key-escrow) 
```

Comment:
"FileVault" 

also, @rachaelshaw  not yet in the handbook but will be tomorrow pending Mike's approval.👇

![image](https://github.com/fleetdm/fleet/assets/108141731/38db7411-846b-47e8-9120-8454a4e72afa)